frontend/components/settings/UsernameChange.vue

```diff
@@ -0,0 +1,147 @@
+ */
+function changeUsername() {
+    emit("changeUsername", newUsername.value);
+    handleReset();
+}
+</script>
+
+<template>
+    <div>
+        <Sidebar
+            v-model:visible="isVisible"
+            modal
+            position="right"
+            :auto-z-index="true"
+            :draggable="false"
+            class="z-50 mt-auto flex h-[95%] w-full flex-col rounded-t-md bg-background font-nunito dark:bg-background-dark sm:hidden sm:w-4/5 md:rounded-xl lg:-z-10"
+            :pt="{
+                root: {
+                    class: 'font-nunito bg-background dark:bg-background-dark z-10 lg:-z-10 lg:hidden ',
+                },
+                header: {
+                    class: 'flex justify-start pb-2 pl-9 font-nunito bg-background dark:bg-background-dark dark:text-natural-50 rounded-3xl',
+                },
+                title: {
+                    class: 'font-nunito text-4xl font-semibold',
+                },
+                content: {
+                    class: 'font-nunito bg-background dark:bg-background-dark px-0 -ml-2 sm:pr-12 h-full',
+                },
+                footer: { class: 'h-0' },
+                closeButton: {
+                    class: 'justify-start w-full h-full items-center collapse',
+                },
+                mask: {
+                    class: 'sm:collapse bg-natural-50',
+                },
+            }"
+            @hide="close"
+        >
+            <template #header>
+                <button class="-ml-6 flex justify-center pr-4" @click="close">
+                    <span class="pi pi-angle-left text-2xl" />
+                </button>
+                <div class="font-nunito text-3xl font-semibold">
+                    <T key-name="dashboard.user.settings.user.name" />
+                </div>
+            </template>
+            <div class="flex h-full flex-col pl-8">
+                <div
+                    class="-pt-4 overflow-hidden overflow-ellipsis text-[0.95rem] text-natural-700 dark:text-natural-200"
+                >
+                    <T
+                        key-name="dashboard.user.settings.change.username.description.part1"
+                    />
+                    <br />
+                    <T
+                        key-name="dashboard.user.settings.change.username.description.part2"
+                    />
+                    <br />
+                    <T
+                        key-name="dashboard.user.settings.change.username.description.part3"
+                    />
+                    <b> {{ username }} </b>
+                </div>
+                <div class="flex items-center pl-6 pt-4">
+                    <div class="flex w-full flex-col items-center">
+                        <div
+                            class="mb-2 mr-10 flex w-full items-start text-[0.95rem] text-text dark:text-natural-200"
+                        />
+                        <input
+                            id="username"
```

Comment:
![image](https://github.com/user-attachments/assets/6ac2c635-66bc-4cbf-a07c-48ab62cae5dc)
can u put the username into the input field too, please

Reply:
Found out Username and Displayname dissapears out of input field after opening it for the second time. Please fix